README.md

```diff
@@ -9,26 +9,20 @@ See https://www.consul.io/docs/ecs for full documentation.
 
 ## Architecture
 
-![Architecture](https://github.com/hashicorp/terraform-aws-consul-ecs/blob/main/_docs/architecture.png?raw=true)
+![Architecture](./_docs/architecture.png?raw=true)
```

Comment:
In order for this README to show up correctly on the [Terraform registry](https://registry.terraform.io/modules/hashicorp/consul-ecs/aws/latest#architecture), this must be an absolute URL. We need to revert this change.

Reply:
We should take this opportunity to update our docs link to use the https://developer.hashicorp.com/consul/docs/ecs URL. See https://github.com/hashicorp/terraform-aws-consul-ecs/pull/198/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R8

Reply:
I just used relative paths to make sure the updated images (present in the feature branch) appear properly when rendered as markdown. I have an action to modify the same (at multiple other places as well) when I raise a giant PR against the main branch. WDYT?